app/src/app/index.jsx

```diff
@@ -33,17 +33,32 @@ import ProfilePage from '$userpages/components/ProfilePage'
 import PurchasesPage from '$userpages/components/PurchasesPage'
 import ProductsPage from '$userpages/components/ProductsPage'
 
-// Docs
+// Docs (depreciated in Aug/Sept 2019)
```

Comment:
Typo, depreciated = deprecated (or else sell now...)

Reply:
👍 
Being schooled on my English skills in my 30s by a non-native speaker is a new low for me 😆